code/modules/projectiles/guns/launcher/bows/bow_messages.dm

```diff
@@ -0,0 +1,52 @@
+/obj/item/gun/launcher/bow/proc/show_string_relax_message(mob/user)
+	if(user)
+		user.visible_message(
+			"\The [user] relaxes the tension on \the [src]'s string.",
+			"You relax the tension on \the [src]'s string."
+		)
+
+/obj/item/gun/launcher/bow/proc/show_unload_message(mob/user)
+	if(user)
+		user.visible_message(
+			"\The [user] removes \the [_loaded] from \the [src].",
+			"You remove \the [_loaded] from \the [src]."
+		)
+
+/obj/item/gun/launcher/bow/proc/show_draw_message(mob/user)
+	if(user)
+		user.visible_message(
+			SPAN_NOTICE("\The [user] begins to draw back the string of \the [src]."),
```

Comment:
all of these cases of `the string` could easily be `\the [string]`, and `\the [src]'s string` could be `\the [src]'s [string.name]`